sql/sys_vars.h

```diff
@@ -840,34 +840,54 @@ class Sys_var_charptr: public sys_var
 
 class Sys_var_version : public Sys_var_charptr
 {
+private:
+  char withsuffix[SERVER_VERSION_LENGTH];
```

Comment:
s/char/uchar/g ?

Reply:
I do not understand. What do you mean?
`char` is on purpose as is minimizes casts for `memcpy` and other.

Reply:
Ack, OK then. My suggestion was in order to remove some of casts with uchar, please keep whichever results in fewer casts.